core/services/relay/evm/ocr2vrf.go

```diff
@@ -81,12 +82,12 @@ func (r *ocr2vrfRelayer) NewDKGProvider(rargs relaytypes.RelayArgs, pargs relayt
 	}, nil
 }
 
-func (r *ocr2vrfRelayer) NewOCR2VRFProvider(rargs relaytypes.RelayArgs, pargs relaytypes.PluginArgs) (OCR2VRFProvider, error) {
+func (r *ocr2vrfRelayer) NewOCR2VRFProvider(rargs relaytypes.RelayArgs, pargs relaytypes.PluginArgs, ethKeystore keystore.Eth) (OCR2VRFProvider, error) {
```

Comment:
I would recommend instead creating an interface specific to this module for keystore that only includes the required methods.